apps/src/templates/studioHomepages/SetUpMessage.jsx

```diff
@@ -48,8 +48,8 @@ const styles = {
   }
 };
 
-const SetUpMessage = React.createClass({
-  propTypes: {
+export default Radium(class SetUpMessage extends Component {
```

Comment:
I _think_ this is the only component in this PR that doesn't have its own test coverage.  Up for a quick unit test?

Reply:
We have coverage on uses of `SetUpMessage` in https://github.com/code-dot-org/code-dot-org/blob/staging/apps/test/unit/templates/studioHomepages/SetUpCoursesTest.jsx and https://github.com/code-dot-org/code-dot-org/blob/staging/apps/test/unit/templates/studioHomepages/SetUpSectionsTest.jsx. Is that sufficient?

Reply:
Yeah, that's great!  I should have waited for the coverage report to be generated. :grin: